BlockSettleUILib/Trading/RFQReplyWidget.cpp

```diff
@@ -267,6 +267,14 @@ void RFQReplyWidget::onPulled(const std::string& settlementId, const std::string
    quoteProvider_->CancelQuoteNotif(QString::fromStdString(reqId), QString::fromStdString(reqSessToken));
 }
 
+void RFQReplyWidget::onUserConnected(const bs::network::UserType &)
```

Comment:
Why user type is not checked here? I believe we shouldn't allow auto-signing or auto-quoting for, say, MD-only users.